foundations/git_basics/git_basics.md

```diff
@@ -13,6 +13,8 @@ By the end of this lesson, you should be able to do the following:
 
 ### Assignment
 
+Note: As of October 1st, 2020, all new Git repositories will create a default branch named 'main' instead of 'master'. (Visit [this article](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main) by Carolyn Heinze if you wish to understand the reasons behind the change.)
```

Comment:
I think just the first sentence would be good. I think we mention elsewhere about the update. So a short note without the link should be good here.

“ Note: As of October 1st, 2020, all new Git repositories will create a default branch named 'main' instead of 'master'.”